src/test/java/gov/healthit/chpl/aqa/stepDefinitions/CollectionsPagesSteps.java

```diff
@@ -48,9 +34,21 @@ public void openAcbFilterOptions() {
      */
     @Given("^I am on \"([^\"]*)\" collections page: \"([^\"]*)\"$")
     public void loadCollectionsPage(final String ptitle, final String pname) {
-      driver.get(url + "#/collections/" + pname);
-      WebDriverWait wait = new WebDriverWait(driver, TIMEOUT);
-      wait.until(ExpectedConditions.visibilityOf(CollectionsPages.mainContent(driver)));
+        getDriver().get(getUrl() + "#/collections/" + pname);
+        WebDriverWait wait = new WebDriverWait(getDriver(), TIMEOUT);
+        wait.until(ExpectedConditions.visibilityOf(CollectionsPages.mainContent(getDriver())));
+    }
+
+    /**
+     * Loads collections pages.
+     * @param pname to get page name in url
+     * @param tEnv test environment in which tests will be run
+     */
+    @Given("^I am on collections page: \"([^\"]*)\" on \"([^\"]*)\"$")
+    public void loadCollectionPage(final String pname, final String tEnv) {
```

Comment:
I don't like having two methods with almost exactly the same name and the same signature, but I don't know what the right solution is